content/06.know-your-problem.md

```diff
@@ -28,7 +28,9 @@ If one assumes that all the samples are independent but that is in fact not the
 
 In general, deep learning has an increased tendency for overfitting, compared to classical methods, due to the large number of parameters being estimated, making issues of adequate sample size even more important (see [Tip 7](#overfitting)).
 For a large dataset, overfitting may not be a concern, but the modeling power of deep learning may lead to more spurious correlations and thus incorrect interpretation of results (see [Tip 9](#interpretation)).
-Finally, it is important to note that with the exception of very specific cases of unsupervised data analysis, it is generally the case that a molecular or imaging dataset does not have much value without appropriate clinical or demographic data; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
+It is important to note that often molecular or imaging datasets require appropriate clinical or demographic data to support robust analyses; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).

```

Comment:
Maybe changing "It is important to note that often molecular or imaging datasets require appropriate clinical"

to 

"It is important to note that molecular or imaging datasets often require appropriate clinical"

will improve readability a bit